README.md

```diff
@@ -212,6 +212,107 @@ You may need to set up multiple automated transfer instances, for example if req
 
 In case different hooks are required for each instance, a possible approach is to checkout a new instance of the automation tools, for example in `/usr/lib/archivematica/automation-tools-2`
 
+Archivematica Client
+--------------------
+
+The transfers/amclient.py script is a module and CLI that provides functionality for interacting with the various Archivematica APIs.
+
+Basic usage: amclient.py <subcommand> [optional arguments] <positional argument(s)>
+  E.g.: amclient.py close-completed-transfers --am-user-name islandora 234deffdf89d887a7023546e6bc0031167cedf6
```

Comment:
Can you modify the example commands so that they are wrapped in backticks so that Markdown will display them as fixed-width code? This will also prevent errors like the fact that `<subcommand>` is not displaying in the rendered Markdown because MD is thinking it's HTML. (You can view the README on GH to see how your Markdown is, or is not, working.)